ai/gpt_finetune/data_process.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import json
+from utils import dicts_to_jsonl
+
+parser = argparse.ArgumentParser(description='This script will process the json data to the right jsonl format.')
+parser.add_argument('-i', '--input', type=str, help='Input json file path')
+parser.add_argument('-o', '--output', type=str, help='Output jsonl file path')
+args = parser.parse_args()
+
+input_file = args.input
+output_file = args.output
+ 
+f = open(input_file, 'r')
+raw_data = json.load(f)
+f.close()
+
+data_list = []
+sys_prompt = "You are the assistant that helps classifying issues in Antrea, a software providing networking and security services for a Kubernetes cluster. Please only output the classification results as some words, which are closed by apostrophes and connected by comma."
```

Comment:
Since the prompt may vary depending on different tasks, we should place it in a separate configuration file.